src/content/en/2019/http2.md

```diff
@@ -39,10 +51,13 @@ Finally, HTTP/2 introduced two new concepts: *header compression* allowed those
 
 ## Adoption of HTTP/2
+As mentioned above, Internet Protocols are often difficult to adopt since they are ingrained into so much of the infrastructure that makes up the internet. This makes introducing any changes slow and difficult. IPv6 for example has been around for 20 years but has [struggled to be adopted](https://www.google.com/intl/en/ipv6/statistics.html).
+
+HTTP/2 however, was different as it was effectively hidden in HTTPS (at least for the browser uses cases), removing barriers to adoption as long as both the browser and server supported it. Browser support has been very strong for some time and the advent of auto updating *evergreen* browsers has meant that an estimated [95% of global users now support HTTP/2](https://caniuse.com/#feat=http2).
 
-As mentioned above, Internet protocols are often difficult to adopt since they are ingrained into so much of the infrastructure that makes up the internet. This makes introducing any changes slow and difficult. IPv6 for example has been around for 20 years but has [struggled to be adopted](https://www.google.com/intl/en/ipv6/statistics.html). HTTP/2 however, was different as it was effectively hidden in HTTPS (at least for the browser uses cases) removing barriers to adoption as long as both the browser and server supported it. Browser support has been very strong for some time and the advent of auto updating *evergreen* browsers has meant that an estimated [95% of global users support HTTP/2 now](https://caniuse.com/#feat=http2). For this Web Almanac we use HTTP Archive which runs a Chrome web crawler on the approximately 5 million top websites (on both Desktop and Mobile with a slightly different set for each). This shows that HTTP/2 usage is now the majority protocol - an impressive feat just 4 short years after formal standardization:
+For this Web Almanac, we used HTTP Archive which runs a Chrome web crawler on approximately 5 million of the top websites on both desktop and mobile, with a slightly different set for each device. Our analysis shows that HTTP/2 usage is now the majority protocol - an impressive feat just 4 short years after formal standardization:
 
-![Timeseries of HTTP/2 Requests](/static/images/2019/20_HTTP_2/http2usage.png)
+![](https://github.com/HTTPArchive/almanac.httparchive.org/raw/master/src/static/images/2019/20_HTTP_2/http2usage.png)
```

Comment:
Yes please revert any of these instances where possible - I missed one of your merges as I didn't do all my editing for this chapter in one sitting.

Reply:
Yeah sorry about that. Should have held off until you were done 🤦‍♂️. It was part of a larger commit so reverting is tricky. Happy for you to force push this and I'll add them back in later if you're cool with that?

Spotted a couple of technical changes I need to make to the HTML to get it to work on mobile anyway but will hold off until this is finished to avoid more merge fun! 😊